src/gui/newwizard/pages/accountconfiguredwizardpage.cpp

```diff
@@ -37,7 +37,7 @@ AccountConfiguredWizardPage::AccountConfiguredWizardPage(const QString &defaultS
         _ui->advancedConfigGroupBox->setVisible(false);
     }
 
-    if (!Theme::instance()->showVirtualFilesOption()) {
+    if (!Theme::instance()->enableExperimentalFeatures() || !Theme::instance()->showVirtualFilesOption()) {
```

Comment:
Tbh, the `enableExperimentalFeatures()` should be moved into the `showVirtualFilesOption()`. Otherwise there are again two places where that is implemented, and the `showVirtualFilesOption` is exactly what it is about, or?

Reply:
This is the case already. Currently, however, `forceVirtualFilesOption` is evaluated later than `showVirtualFilesOption`. "force" seems higher prio than "show". We can change this, though.